src/test/kotlin/day04/EnemyTest.kt

```
@@ -0,0 +1,36 @@
+    lateinit var enemy: Enemy
+
+    @Before
+    fun setUp() {
+        hero = Hero("hero",5, Sword("common sword", 5))
+        wizard = Wizard("wizard", 10, 5)
+        enemy = Enemy("enemy")  // hp = MAX_HP(20)
+    }
+
+    @After
+    fun tearDown() {
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)